lib/SILOptimizer/Utils/Devirtualize.cpp

```diff
@@ -121,7 +121,7 @@ static bool isEffectivelyFinalMethod(FullApplySite AI,
   if (!CHA)
     return false;
 
-  // This is a private or a module internal class.
```

Comment:
It's probably a good idea to just drop this and change the `it` in the paragraph below to refer to `this class`.  The picture is more complex than just being public.